stubs/erlang-httpc/run

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env escript
+%% -*- erlang -*-
+ 
+%% trytls (https://github.com/ouspg/trytls) stub for erlang
+ 
+%% Uncomment the following line if needed. Hard to say if compiling is worthwhile here.
+%% -mode(compile).
+ 
+ 
+ 
+main([Url, Port]) ->
+    start_deps(),
+    Result = httpc:request("https://" ++ Url ++ ":" ++ Port),
+    check(Result);
+ 
+main([Url, Port, CaFile]) ->
+    start_deps(),
+    Result = httpc:request(head, {"https://" ++ Url ++ ":" ++ Port, []}, [{ssl,[{cacertfile, CaFile}]}], []),
+    check(Result);
+ 
+main(_) ->
+    usage().
+ 
+usage() ->
```

Comment:
Fatal errors (Not UNSUPPORTED/ACCEPT/REJECT) should exit with a non-zero exit value. See [Return values](https://github.com/ouspg/trytls/blob/master/stubs/README.md#return-values)